homeassistant/components/zha/api.py

```diff
@@ -340,15 +232,15 @@ def async_load_api(hass, application_controller, listener):
     async def websocket_entities_by_ieee(hass, connection, msg):
         """Return a dict of all zha entities grouped by ieee."""
         entities_by_ieee = {}
-        for ieee, entities in listener.device_registry.items():
+        for ieee, entities in zha_gateway.device_registry.items():
```

Comment:
I'd like us to rename `entities` to eg `entity_refs`, to avoid thinking that these are actual entities.

Reply:
This PR or in a quick followup?

Reply:
Ok, let'd do it in the follow up. :+1: